modules/gateways/Blockonomics/Blockonomics.php

```diff
@@ -308,6 +308,7 @@ public function createOrderTableIfNotExist() {
 							$table->integer('bits_payed');
 							$table->string('blockonomics_currency');
 							$table->primary('addr');
+							$table->string('order_currency');
```

Comment:
Does not work for existing users. createOrderTableIfNotExist() does not update blockonomics_orders table during upgrades. An example of performing table upgrades can be seen here:
https://github.com/blockonomics/whmcs-bitcoin-plugin/blob/2373690f0b0fed963f13276ffb6fd405b8255d98/modules/gateways/Blockonomics/Blockonomics.php#L217-L221